monitoring/tls/prom-alertendpoint-host-https.yaml

```diff
@@ -0,0 +1,9 @@
+prometheus:
+  prometheusSpec:
+    alertingEndpoints:
+      - name: v4m-alertmanager 
+        namespace: monitoring
```

Comment:
We can't hard-code the namespace here.  Users can deploy our components into any namespace.